### DISCUSSION TITLE
本地源码启动问题

### DISCUSSION THREAD

**Discussion by zhaosanmei** [langgenius/dify#5358] [Help]
(dify) lizhen@localhost api % flask db upgrade
None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used.
zsh: segmentation fault  flask db upgrade  
pip install 安装包之后，用flask db upgrade启动，提示上面内容。数据库无法创建。
<img width="340" alt="image" src="https://github.com/langgenius/dify/assets/11640365/c5a8a458-3df4-4e20-908b-ad0dbee4bf8f">
系统配置是版本 10.15.7
macOS Catalina

**Comment by lzj1991** [comment]
大佬  解决了吗
